src/app/components/work-item-quick-add/work-item-quick-add.component.ts

```diff
@@ -106,6 +111,53 @@ export class WorkItemQuickAddComponent implements OnInit, OnDestroy, OnChanges,
     this.workItem.relationships = new WorkItemRelations();
     this.workItem.type = 'workitems';
     this.workItem.attributes['system.state'] = 'new';
+    //set the default work item type
+    this.workItem.relationships.baseType = {
+      data: {
+        id: this.selectedType?this.selectedType.id:'testtypeid',
+        type: this.selectedType?this.selectedType.type:'testtype'
+      }
+    }
+    //Go through the active filters and apply them
+    let currentFilters = this.filterService.getAppliedFilters();
+    if( currentFilters.length ) {
+      for (let f=0; f < currentFilters.length; f++) {
+        switch (currentFilters[f].id) {
+          case 'iteration':
+            this.workItem.relationships.iteration = {
+              data: {
+                id: currentFilters[f].value,
+                type: currentFilters[f].id
+              }
+            } as any;
+            break;
+          case 'area':
+            this.workItem.relationships.area = {
+              data: {
+                id: currentFilters[f].value,
+                type: currentFilters[f].id
+              }
+            } as any;
+            break;
+          case 'workitemtype':
+            this.workItem.relationships.baseType = {
+              data: {
+                id: currentFilters[f].value,
+                type: currentFilters[f].id+'s'
+              }
+            } as any;
+            break;
+          case 'assignee':
+            this.workItem.relationships.assignees = {
+              data: [{
+                id: currentFilters[f].value,
+                type: 'identities'
+              }]
+            } as any;
```

Comment:
Work item gets created in the context of assignee but it will not appear in the filtered list.